src/components/collections/Footer/component.js

```diff
@@ -122,6 +124,7 @@ Component.defaultProps = {
   currencyValue: null,
   languageOptions: [],
   languageValue: null,
+  navigationItems: [],
```

Comment:
```suggestion
  navigationItems: null,
```
I don't think we need the array for anything?